Gaijinizer patch/Map057.txt

```diff
@@ -40,8 +41,9 @@ I can not ignore the existence of that Nina demon
 > CONTEXT: Map057/events/5/pages/0/28/Dialogue/yuuma
 > CONTEXT: Map057/events/5/pages/1/29/Dialogue/yuuma
 > CONTEXT: Map057/events/5/pages/2/28/Dialogue/yuuma
-And behind the park, the army set up a meeting
-point. The campsite should be left almost intact.#MTLed
+And that park route also has a meeting point set
+up by the military. The campsite should still be
+mostly intact.#Ron
```

Comment:
Minor flow:
>_**The park route also has a meeting point set
up by the military.**_

Reply:
Think you missed this one?

Reply:
That one's done isn't it? The file is now

> The park route also has a meeting point set
up by the military. The campsite should still be
mostly intact.